streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -588,6 +592,29 @@ public void testCloseIsIdempotent() {
             closeCount, MockMetricsReporter.CLOSE_COUNT.get());
     }
 
+    @Test
+    public void testAddThread() {
```

Comment:
I would prefer to use `shouldAddThread()` as name although the pattern is different for the other test methods.